src/API/Base.php

```diff
@@ -460,9 +499,9 @@ public static function update_merchant_feed( $merchant_id, $feed_id, $args ) {
 	 *
 	 * @return mixed
 	 */
-	public static function get_merchant_feed( $merchant_id, $feed_id ) {
+	public static function get_merchant_feed_report( $merchant_id, $feed_id ) {
 		return self::make_request(
-			"catalogs/datasource/feed_report/{$merchant_id}/",
+			'catalogs/datasource/feed_report/' . $merchant_id . '/',
```

Comment:
This should revert back to interpolation.

```suggestion
			"catalogs/datasource/feed_report/{$merchant_id}/",
```